db/seeds.rb

```diff
@@ -10,80 +10,167 @@
 
 require "faker"
 
-email_providers = ["gmail.com", "yahoo.com", "hotmail.com", "outlook.com", "aol.com"]
+user_names = ["user", "user1", "user2", "user3"]
 
-# Create multiple users
-5.times do
-  name = Faker::Name.first_name
-  user = User.create!(
-    first_name: name,
+# Create User for Caley workspace
+email = "user@caley.to"
+user = User.find_or_create_by!(email:) do |user|
+  user.assign_attributes({
+    first_name: "user".capitalize,
     last_name: Faker::Name.last_name,
-    email: name,
+    email: "user",
     password: "password",
     password_confirmation: "password"
-  )
-
-  puts "Loading seed data for user: #{user.email}"
+  })
+end
+puts "Created User for Caley workspace - #{user.email}"
 
-  # Create a workspace for each user
-  workspace = Workspace.create!(
-    name: "#{Faker::Company.name} Workspace",
+# Create Caley workspace
+name = "Caley"
```

Comment:
Small detail, but would you make the workspace names and team names constants and put them at the top of the file? I'd like to make it clear in the future that these shouldn't be changed
Ex: `WORKSPACE_NAME = "Caley".freeze`